README.md

```diff
@@ -14,20 +40,66 @@ IRC channel: [freenode](https://freenode.net) #googlebenchmark
 
 [Assembly Testing Documentation](docs/AssemblyTests.md)
 
+## Requirements
+
+Google Benchmark uses C++11 when building the library. As such we require
+a modern C++ toolchain, both compiler and standard library.
+
+The following minimum versions are strongly recommended build the library:
+
+* GCC 4.8
+* Clang 3.4
+* Visual Studio 2013
+* Intel 2015 Update 1
+
+Anything older *may* work.
```

Comment:
I don't think they will?
These are minimal versions, not minimal recommended.

Reply:
Will remove the last line. Thanks.